downstreamadapter/dispatchermanager/event_dispatcher_manager.go

```diff
@@ -165,7 +165,7 @@ func NewEventDispatcherManager(
 	}
 
 	var err error
-	manager.sink, err = sink.NewSink(ctx, manager.config, manager.changefeedID)
+	manager.sink, err = sink.NewSink(ctx, manager.config, manager.changefeedID, manager.errCh, manager.pdClock)
```

Comment:
We can omit these two variables in the general newSink.
1. for pdClock, we can get directly by ```pdClock := appcontext.GetService[pdutil.Clock](appcontext.DefaultPDClock)```
2. for errCh we can deal with as `mysqlSink or KafkaSink`